packages/space-opera/src/components/camera_settings/camera_settings.ts

```diff
@@ -106,8 +105,13 @@ export class CameraTargetInput extends ConnectedLitElement {
   @property({attribute: false}) change?: (newValue: Vector3D) => void;
   @internalProperty() target?: Vector3D;
 
+  // @ts-ignore
   stateChanged(state: State) {
-    this.target = getCamera(state).target ?? getInitialCamera(state).target;
+    if (getModelViewer() !== undefined && getModelViewer() !== null) {
```

Comment:
This is the one time where `==` is called for; we use `getModelViewer() != null` to catch both of these very regularly.